packages/shopware-6-client/__tests__/services/CartService/updateLineItem.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { updateLineItem } from "@shopware-pwa/shopware-6-client";
+import { apiService } from "../../../src/apiService";
+
+jest.mock("../../../src/apiService");
+const mockedAxios = apiService as jest.Mocked<typeof apiService>;
+
+describe("CartService - addLineItem", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it("should call valid endpoint and return cart with no items", async () => {
+    mockedAxios.patch.mockResolvedValueOnce({
+      data: {
+        name: "3a64e872ca404522a2c5d43ebc751e6b",
+        token: "NWDdcRTTWoPk4Ngv13z5NDMMsDFRb9W6",
+        lineItems: [
+          {
+            id: "geawq90a5dab4206843d0vc3sa8wefdf",
+            label: "Red T-Shirt",
```

Comment:
you can go ahead with faker in a place like this
```
import { commerce } from "faker"
```

```suggestion
            label: commerce.productName(),
```

all the id could be generated by faker as well

Reply:
niiiiice! ;D